### PR TITLE
Fix cloud report python3

### PR DESCRIPTION
**Pull request by bentsi**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [ ] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/utils/cloud_monitor/report.py [hunk 1]
```diff
@@ -30,7 +30,8 @@ def to_html(self):
         return self.render_template()
 
     def to_file(self):
-        with tempfile.NamedTemporaryFile(prefix='cloud-report_', delete=False, suffix='.html') as report_file:
+        with tempfile.NamedTemporaryFile(mode="w", encoding="utf-8",
```
Comment (yarongilor) [inline sdcm/utils/cloud_monitor/report.py]: the name of this method is not completely clear.

Comment (fruch) [reply sdcm/utils/cloud_monitor/report.py]: It's a built-in, a bit hard to change it :)

**Comment by fruch** [comment]
Why backports ? Those branches don't have python3 in them ?

**Comment by bentsi** [comment]
@fruch backport was intended for the commit "Fixing a bug in verbose mode of list_instances_aws" :)

**Pull request merged**